### ISSUE TITLE
Missing Mob Spawn Time Interval

### ISSUE THREAD

**Issue by dowzer** [EdenServer/community#5985]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [x] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

When a zone loads, every monster will appear to spawn at an exact time. Let's use 00:17 as an example.

Each time a monster is killed, the next respawn that occurs will be at that exact time or 30 seconds after it. If the respawn is before 00:17, it will spawn at 00:17 - if the respawn is slightly after 00:17 - it will instead spawn 30 seconds later at 00:47.

All pops until that zone server goes down will be at xx:yy:17 or xx:yy:47. 

https://www.bluegartr.com/threads/89506-Timing-mob-NM-respawns

This would be interesting to see if the interval still exists on current retail. [I believe it does as the same testing was done on high level capacity point monsters that have a very low 60 second respawn. ](https://www.ffxiah.com/forum/topic/29127/random-question-thread-ffxi-related/855/#3712797)

It would still be a good idea to test on retail servers anyway. I'm only basing this from what I've read, it could have other variances that I'm not aware of without testing.

**Comment by dowzer** [comment]
https://www.youtube.com/watch?v=t1HA-tmzfdE

Confirmation that spawn mechanics still follow this interval.

**Comment by dowzer** [comment]
Also need to note that each cluster or zone has a different time, possibly due to when they come online.

KRT was :24 and :54 while the Boyahda Tree was :17  and :47

**Comment by 59blargedy** [comment]
resolved by another dev in July 13, 2026 update

**Comment by dowzer** [comment]
Due to an overwhelming amount of upsetti-spaghetti, or otherwise known as sensitive HNM linkshell players, I've looked into this further. I had doubts that Fafnir/Nidhogg, Adamantoise/Aspidochelone, or Behemoth/King Behemoth were controlled by the server clock. Not only from personal experience during that era, but also from video in era, and a very infamous developer. 

I'm confident that every player who camped kings in the past never had to kill a fodder mob to establish the exact spawn time, I'm positive everyone will agree with that.

Mafai developed Arenz, an alliance based packet based program specifically for kings, if you catch my drift. I believe his experience over the single person claiming he is wrong in the thread I attached.
<img width="1491" height="631" alt="Image" src="https://github.com/user-attachments/assets/c967ade2-ba8e-405a-b733-8a729f5c6f23" />

A player references :58:28 and it pops specifically at that time: 
https://www.youtube.com/watch?v=Njjox9oII7w

A player counts down and it pops at that time:
https://www.youtube.com/watch?v=un2kC-AVLCs

**Comment by DeuceFFXI** [comment]
> Due to an overwhelming amount of upsetti-spaghetti, or otherwise known as sensitive HNM linkshell players, I've looked into this further. I had doubts that Fafnir/Nidhogg, Adamantoise/Aspidochelone, or Behemoth/King Behemoth were controlled by the server clock. Not only from personal experience during that era, but also from video in era, and a very infamous developer. 
> 
> I'm confident that every player who camped kings in the past never had to kill a fodder mob to establish the exact spawn time, I'm positive everyone will agree with that.
> 
> Mafai developed Arenz, an alliance based packet based program specifically for kings, if you catch my drift. I believe his experience over the single person claiming he is wrong in the thread I attached.
> <img width="1491" height="631" alt="Image" src="https://github.com/user-attachments/assets/c967ade2-ba8e-405a-b733-8a729f5c6f23" />
> 
> A player references :58:28 and it pops specifically at that time: 
> https://www.youtube.com/watch?v=Njjox9oII7w
> 
> A player counts down and it pops at that time:
> https://www.youtube.com/watch?v=un2kC-AVLCs

I have to agree that it's highly unlikely for HNMs with windows to follow/have followed the server time rule.  It makes little sense when you consider the fact their spawn mechanics were different from nearly every other NM in the game.  I'm not sure how much information was gathered regarding the spawn mechanics of NMs, but I believe this warrants a deeper look into the change overall if it was only based on a handful of NMs.

**Comment by dowzer** [comment]
For retail testing purposes, Tiamat,/Wyrms Cerberus/ToAU HNM, King Arthro, and Zilart Beastmen King timers weren't reduced and still respawn according to era. My hypothesis is that only HNM with 30 min or 60 min windows spawn independently of the server clock. While far more tedious, this could be verified by testing if Zilart Beastmen Kings spawn by the server clock.

**Comment by InoUno** [comment]
> For retail testing purposes, Tiamat,/Wyrms Cerberus/ToAU HNM, King Arthro, and Zilart Beastmen King timers weren't reduced and still respawn according to era. My hypothesis is that only HNM with 30 min or 60 min windows spawn independently of the server clock. While far more tedious, this could be verified by testing if Zilart Beastmen Kings spawn by the server clock.

I went on retail and camped one of the Zilart Kings, and they do indeed *not* follow the 30 second interval spawning of other mobs. It spawned based on the time the mob despawned the previous day, as the Mafai guy also claimed.

This will be fixed for HNMs in the next update.
